design/build.gradle.kts

```diff
@@ -5,7 +5,7 @@ plugins {
 
 android {
     namespace = "com.rjnr.design"
-    compileSdk = ConfigurationData.compileSdk
+    compileSdk = 34
```

Comment:
the old ways its much better, can you revert this change here and in the other modules?